### PR TITLE
Open docs in new tab

### PR DESCRIPTION
**Pull request by microsoftsam**
Open docs in new page rather than the side docs.

### CODE REVIEW

File: webapp/src/projects.tsx [hunk 1]
```diff
@@ -145,12 +145,14 @@ export class Projects extends data.Component<ISettingsProps, ProjectsState> {
...
                 default:
-                    const m = /^\/#tutorial:([a-z0A-Z0-9\-\/]+)$/.exec(scr.url);
+                    const m = /^\/#tutorial:([a-z0A-Z0-9\-\/]+)$/.exec(scr.url); // Tutorial
                     if (m) this.props.parent.startTutorial(m[1]);
                     else {
-                        if (scr.youTubeId && !scr.url)
+                        if (scr.youTubeId && !scr.url) // Youtube video
                             window.open('https://youtu.be/' + scr.youTubeId, 'yt');
-                        else if (/^https:\/\//i.test(scr.url))
+                        else if (/^https:\/\//i.test(scr.url)) // External video
```
Comment (pelikhan) [inline webapp/src/projects.tsx]: both paths do the same... we should allow local urls (start with / or https urls only)

File: webapp/src/projects.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ export class Projects extends data.Component<ISettingsProps, ProjectsState> {
                             window.open('https://youtu.be/' + scr.youTubeId, 'yt');
                         else if (/^https:\/\//i.test(scr.url)) // External video
                             window.open(scr.url, '_blank');
-                        else if (scr.url) // Docs url, open in new tab
+                        else if (/^\//i.test(scr.url)) // Docs url, open in new tab
                             window.open(scr.url, '_blank');
                         else
                             this.props.parent.newEmptyProject(scr.name.toLowerCase(), scr.url);
```
Comment (pelikhan) [inline webapp/src/projects.tsx]: probably just want to show a warning and ignore the URL.

File: webapp/src/projects.tsx (hunk 1 above)
Comment (microsoftsam) [reply webapp/src/projects.tsx]: Fixed

File: webapp/src/projects.tsx (hunk 2 above)
Comment (microsoftsam) [reply webapp/src/projects.tsx]: Cool. Done

**Comment by microsoftsam** [comment]
@pelikhan ping for another look

File: webapp/src/projects.tsx [hunk 3]
```diff
@@ -152,8 +152,11 @@ export class Projects extends data.Component<ISettingsProps, ProjectsState> {
...
                             window.open(scr.url, '_blank');
-                        else if (/^\//i.test(scr.url)) // Docs url, open in new tab
-                            window.open(scr.url, '_blank');
+                        else if (scr.url) // Docs url, open in new tab
+                            if (/^\//i.test(scr.url))
+                                window.open(scr.url, '_blank');
+                            else
+                                core.errorNotification(lf("Sorry, the project url looks invalid."));
                         else
                             this.props.parent.newEmptyProject(scr.name.toLowerCase(), scr.url);
```
Comment (pelikhan) [inline webapp/src/projects.tsx]: scr.url should always be undefined here

**Pull request merged**